lib/timeline/component/item/Item.js

```diff
@@ -546,9 +544,6 @@ class Item {
             remove: this.data.editable
           }
         } else if (typeof this.data.editable === 'object') {
-          // TODO: in timeline.js 5.0, we should change this to not reset options from the timeline configuration.
-          // Basically just remove the next line...
-          this.editable = {};
```

Comment:
This is the only change I'm uncertain about - Though this change is overdue, fixing this bug technically constitutes a breaking change, & possibly should be deferred until a major release?